lib/absinthe/middleware/telemetry.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Absinthe.Middleware.Telemetry do
+  @moduledoc """
+  Gather and report telemetry about an individual resolver function
+  """
+  @telemetry_event [:absinthe, :resolver]
```

Comment:
This should probably be `[:absinthe, :resolved]` since that sounds more like an event.

Reply:
After reviewing GraphQL spec, how about `[:absinthe, :resolve, :field]`

Reply:
👍